docs/staticAnalysis.md

```diff
@@ -25,14 +25,23 @@ Conversely, when updating any tool, ensure that the tool version is supported by
 [CheckStyle](http://checkstyle.sourceforge.net/index.html) helps to enforce coding standard in Java source code.
 The rules to be used are configured in a ruleset file; in TEAMMATES the file can be found [here](../static-analysis/teammates-checkstyle.xml).
 The plugin for Eclipse can be found [here](http://eclipse-cs.sourceforge.net/#!/).
+The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/plugin/1065-checkstyle-idea).
```

Comment:
> There is a more popular and higher rated PMD plugin called 'QA Plug - PMD'

Yes, I tried that before the one I ended up using but wasn't able to get it working. Are you using that for your other projects?